.circleci/config.yml

```diff
@@ -469,12 +482,16 @@ jobs:
     working_directory: ~/aeternity
     steps:
       - checkout
+      - *restore_macos_package_cache
       - run:
           name: Install required tools
           command: |
-            brew update
```

Comment:
I see this removes [`update`](https://docs.brew.sh/Manpage#update-options) while it is recommended in [the documentation](https://github.com/aeternity/aeternity/blame/86430a4cfe0ff773f709a5b24056bbd64b1d09a0/docs/installation.md#L73-L74). This may cause users to spot issues before our CI.

Reply:
I saw that every `brew install` command invokes update, that's why i removed it

Reply:
Could you please add your rationale in the CI config file, or at least in the PR.
I see myself having the same doubt when seeing that CI code path again...

Reply:
I brought it back, as it is more clear what it's doing